scripts/test_runner.sh

```diff
@@ -52,7 +52,7 @@ if [ -n "${TEST_DIR}" ]; then
         fi
     elif [ "${TEST_DIR}" = "versionbump" ]; then
        declare -a OPTS
-          OPTS=(-timeout=65m -v -count=1 ./entrypoint/versionbump/... -tags="${TEST_TAG}")
+          OPTS=(-timeout=45m -v -count=1 ./entrypoint/versionbump/... -tags=versionbump)
```

Comment:
let's leave this as it was: `OPTS=(-timeout=65m -v -count=1 ./entrypoint/versionbump/... -tags="${TEST_TAG}")`